src/main/scala/org/allenai/wikitables/TestWikiTablesCli.scala

```diff
@@ -57,22 +59,29 @@ class TestWikiTablesCli extends AbstractCli() {
     loader.done()
 
     // Read test data.
-    val testData = options.valuesOf(testDataOpt).asScala.flatMap(filename => {
-      WikiTablesUtil.loadDataset(filename, false, null, options.valueOf(maxDerivationsOpt))
-    })
+    val buffer = ListBuffer[(WikiTablesExample, RawEntityLinking)]()
+    for (filename <- options.valuesOf(testDataOpt).asScala) {
```

Comment:
Noted. This code is actually obsolete (I have another PR coming), so I've fixed it there.